src/volumetric/volumetric_convex.rs

```diff
@@ -137,6 +178,33 @@ pub unsafe fn convex_mesh_mass_properties<P, I>(convex_mesh: &TriMesh<P>,
     (volume * density, com, itot * density)
 }
 
+#[allow(non_snake_case)]
+pub unsafe fn convex_mesh_mass_properties2D<P>(convex_mesh: &Polyline<P>,
+                                                  density:     <P::Vect as Vect>::Scalar)
+                                                  -> (<P::Vect as Vect>::Scalar, P, <P::Vect as Vect>::Scalar)
```

Comment:
Ditto (`for non_snake_case`). Also, the function arguments and `->` should be vertically aligned.